### PR TITLE
changes to handle multibranch pipeline repo details

### PR DESCRIPTION
**Pull request by AparnaSaripaka**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -132,6 +132,11 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```
Comment (jragula-zs) [inline pom.xml:135]: Do we need to have this dependency now as we are not using any libraries?

File: pom.xml [hunk 2]
```diff
@@ -24,7 +24,7 @@
     <version>2.0.1-SNAPSHOT</version>
     <packaging>hpi</packaging>
     <properties>
-        <jenkins.version>2.319</jenkins.version>
+        <jenkins.version>2.319.1</jenkins.version>
```
Comment (jragula-zs) [inline pom.xml:27]: Is this version change necessary for this feature?

File: pom.xml (hunk 1 above)
Comment (AparnaSaripaka) [reply pom.xml:135]: Yes this is required to extract commit id details

File: pom.xml (hunk 2 above)
Comment (AparnaSaripaka) [reply pom.xml:27]: This is just to run in local. I will revert it.

**Review by jragula-zs — APPROVED** [review]

**Pull request merged**